package.json

```diff
@@ -48,8 +48,7 @@
     "extraResources": "node_modules/@zen/zen-node",
     "appId": "com.zenprotocol.wallet",
     "publish": {
-      "provider": "github",
-      "publishAutoUpdate": false
```

Comment:
@somdoron added this in the [commit for installers through CI](https://github.com/zenprotocol/zenwallet/commit/95253ed05d0ce20199e1922accabec1370ddfd28).

Is it safe to remove it?

@somdoron do you see any reason not to remove it?

Reply:
This was not intended so great catch. will restore. I set it to true when trying to use the auto-update lib and then inadvertently removed it :/. Will restore as there was no reason for me to remove.